### ISSUE TITLE
pyright command line doesn't display any error

### ISSUE THREAD

**Issue by ubitux** [microsoft/pyright#3429]
**Describe the bug**
The `pyright` command line doesn't display any error when it should. The LSP server doesn't seem affected by this issue.

Reproducible with 1.1.244.

*Edit*: apparently it works as expected when using `pyright --lib`. I guess that's on purpose?
*Edit 2*: note that for some strange reason, just running `pyright` without any argument **does** spot the error -- I am extremely confused

**To Reproduce**

Given the following tree/files:
```
.
├── my_python_module
│   ├── my_python_module
│   │   └── __init__.py
│   └── setup.py
└── tests
    └── my_python_test.py
```

`my_python_module/my_python_module/__init__.py`:
```python
class MyPythonClass:
    def my_method(self):
        print("calling my Python method")
```

`my_python_module/setup.py`:
```python
from setuptools import setup

setup(
    name='my_python_module',
    packages=['my_python_module'],
)
```

`tests/my_python_test.py`:
```python
import my_python_module as mpm

c = mpm.MyPythonClass()
c.my_method()

c = mpm.MyNonExistingPythonClass()
c.my_non_existing_method()
```

Running `pyright` after setup:
```
$ python -m venv venv
$ . venv/bin/activate
(venv) $ pip install ./my_python_module
[...]
(venv) $ pyright tests/my_python_test.py
No configuration file found.
No pyproject.toml file found.
stubPath /home/ux/sandbox/pyright/python-test/typings is not a valid directory.
Assuming Python platform Linux
Searching for source files
Found 1 source file
0 errors, 0 warnings, 0 informations
Completed in 0.601sec
```

**Expected behavior**

Within my editor with LSP enabled (neovim), the error is correctly spotted:

![2022-05-05-121327-RioRai8y](https://user-images.githubusercontent.com/34467/166903663-b9e005e8-35d8-471e-8e07-5ad408c2a1f5.png)

*Edit*: it seems `useLibraryCodeForTypes` is set to `true` by default in the LSP config

**Comment by erictraut** [comment]
If you are using `useLibraryCodeForTypes` in your LSP and want to see the same output in the CLI, you will need to pass `--lib`.

**Comment by ubitux** [comment]
But why `pyright` without specifying a file seems to enable `useLibraryCodeForTypes`?

**Comment by erictraut** [comment]
Ah, I missed that. I'll investigate.

**Comment by erictraut** [comment]
I cannot repro the problem as specified. When I recreate the files above and run `pyright` from the root directory of the project, I receive the following output:
```
No configuration file found.
No pyproject.toml file found.
stubPath /Users/eric/Desktop/Test4/typings is not a valid directory.
Assuming Python platform Darwin
Searching for source files
Found 3 source files
/Users/eric/Desktop/Test4/tests/my_python_test.py
  /Users/eric/Desktop/Test4/tests/my_python_test.py:3:9 - error: "MyPythonClass" is not a known member of module (reportGeneralTypeIssues)
  /Users/eric/Desktop/Test4/tests/my_python_test.py:6:9 - error: "MyNonExistingPythonClass" is not a known member of module (reportGeneralTypeIssues)
2 errors, 0 warnings, 0 informations 
Completed in 0.651sec
```

I get the same output when running `pyright --lib`.

I suspect that you're running the CLI from a different directory because your output indicates that it found only one code file when there are clearly three ".py" files in your directory hierarchy.

**Comment by ubitux** [comment]
OK I must have missed something because I'm now unable to reproduce as well. Sorry for the noise and thanks for your time.
